src/components/PostcodeSearch/index.js

```diff
@@ -23,6 +24,14 @@ function PostcodeSearch() {
     }
   };
 
+  const handleRadiusChange = (e) => {
+    const {value} = e.currentTarget
+    
+    setRadius(value);
+    const valueKm = parseInt(value) * 1000;
+    setRadiusOnContext(valueKm);
```

Comment:
Happy to receive any feedback on how to improve the interaction between this form and the GeoContext